custom/reach/templates/reach/reports/program_overview.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'reach/dashboard.html' %}
+{% load hq_shared_tags %}
+{% load menu_tags %}
+
+{% block content %}
+    <div id="program-overview" class="program-overview" >
+        <div class="indicators" data-bind="foreach: sections">
+            <div class="row" data-bind="foreach: $data">
+                <div class="col-md-4 no-padding" >
+                    <div style="padding: 15px;" data-bind="if: format==='numeric'">
+                        <div class="col-md-12 no-padding shadow">
+                            <div class="col-md-12 indicator">
+                                <div class="main-value" data-bind="style: {color: color}"><strong><span data-bind="text: reachUtils.toIndiaFormat(numerator)"></span></strong></div>
+                                <div class="line"><hr></div>
+                            </div>
+                            <div class="col-md-12 indicator-bottom" data-bind="style: {background: color}">
+                                <div><span data-bind="text: indicator"></span></div>
+                                <div><span data-bind="text: diffBetweenMonths()"></span></div>
+                            </div>
+                        </div>
+                    </div>
+                    <div style="padding: 15px;" data-bind="if: format==='percent'">
+                        <div class="col-md-12 no-padding shadow">
+                            <div class="col-md-12 indicator">
+                                <div class="main-value" data-bind="style: {color: color}"><strong><span data-bind="text: percentFormat()"></span></strong></div>
+                                <div class="second-value"><span data-bind="text: reachUtils.toIndiaFormat(numerator)"></span> / <span data-bind="text: reachUtils.toIndiaFormat(denominator)"></span></div>
+                                <div class="line"><hr></div>
+                            </div>
+                            <div class="col-md-12 indicator-bottom" data-bind="style: {background: color}">
+                                <div><span data-bind="text: indicator"></span></div>
+                                <div><span data-bind="text: diffBetweenMonths()"></span></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock content %}
+
+{% block js-inline %}
+    {{ block.super }}
+    <script src="{% static 'reach/js/reports/program_overview.js' %}"></script>
+    <script>
+        $(function () {
+            var programOverviewModel = hqImport('reach/js/reports/program_overview').programOverviewModel();
+            $('#reach-dashboard').koApplyBindings(programOverviewModel);
+            programOverviewModel.callback();
+        })
+    </script>
```

Comment:
Can this be put into a separate json file? Why can't it be included in `program_overview.js`?

Also why is this in `js-inline` instead of `js`

Reply:
Confirming this should be in a separate js file.

Reply:
I moved this to the ```program_overview.js``` file